src/command.lisp

```diff
@@ -8,15 +8,26 @@
 (defvar *this-command*
   "The command now being executed.")
 
+(defvar *this-command-keys* '()
+  "List containing the key sequence that invoked the present command.")
+
+(defvar *current-prefix-arg* nil
+  "The raw prefix argument for the current command.")
```

Comment:
We call this a universal-argument, so I would avoid adding a new term, prefix-arg.
How about the name `*universal-argument*` and `(universal-argument-of-last-command)` ?
If you have any other good names, please let me know.

Reply:
Thanks for your comment!
Yeah, "naming" is one of the reasons I wanted to ask you because we don't need to have the same name as Emacs has.
I'll change the variable to have `universal-argument` in its name.

> It may be a good idea to wrap this variable in a function as well for future refactoring.

Actually, I'm not sure whether it should be wrapped with a function.
If it's exposed as a special variable, it could be convenient when people want it to be changed in a lexical scope, like:

```common-lisp
(let ((*universal-argument* 10))
  (call-command 'something nil))
```

To be clear, I'm not confident with it, so I'd like to hear if there's some kind of rule (or habit) in Lem to do, or if you prefer it for some reason.

Reply:
It could be a reason to wrap by a function that the variable is used differently in some contexts.
For example, as you mentioned, `universal-argument-of-last-command` implies there's also `this-universal-argument` or so.

Reply:
>  If it's exposed as a special variable, it could be convenient when people want it to be changed in a lexical scope, like:

In this case, the second argument of call-command is universal-argument, so binding of the special variable appears unnecessary.
Wrapping a function makes it easier to insert breaks at the point of call, and I believe it increases maintainability.

Reply:
Alright, at least in my use case, it's not necessary to be a special variable, so I'll go with a function style.